test/plugin_functional/plugins/kbn_tp_embeddable_explorer/public/np_ready/public/plugin.tsx

```diff
@@ -0,0 +1,112 @@
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CoreSetup, CoreStart, Plugin } from 'src/core/public';
+import { start } from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public/legacy';
+
+import {
+  Start as InspectorStartContract,
+  Setup as InspectorSetupContract,
+} from '../../../../../../../src/plugins/inspector/public';
+
+import { Plugin as EmbeddablePlugin, CONTEXT_MENU_TRIGGER } from './embeddable_api';
+
+const REACT_ROOT_ID = 'embeddableExplorerRoot';
+
+import {
+  HelloWorldAction,
+  SayHelloAction,
+  SendMessageAction,
+  HelloWorldEmbeddableFactory,
+  ContactCardEmbeddableFactory,
+  HELLO_WORLD_ACTION_ID,
+} from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public/lib/test_samples';
+import { App } from './app';
+
+export interface SetupDependencies {
+  embeddable: ReturnType<EmbeddablePlugin['setup']>;
+  inspector: InspectorSetupContract;
+  __LEGACY: {
+    SavedObjectFinder: React.ComponentType<any>;
+    ExitFullScreenButton: React.ComponentType<any>;
+  };
+}
+
+interface StartDependencies {
+  embeddable: ReturnType<EmbeddablePlugin['start']>;
+  inspector: InspectorStartContract;
+  __LEGACY: {
+    SavedObjectFinder: React.ComponentType<any>;
+    ExitFullScreenButton: React.ComponentType<any>;
+    onRenderComplete: (onRenderComplete: () => void) => void;
+  };
+}
+
+export type EmbeddableExplorerSetup = void;
+export type EmbeddableExplorerStart = void;
+
+export class EmbeddableExplorerPublicPlugin
+  implements
+    Plugin<EmbeddableExplorerSetup, EmbeddableExplorerStart, SetupDependencies, StartDependencies> {
+  public setup(core: CoreSetup, setupDeps: SetupDependencies): EmbeddableExplorerSetup {}
+
+  public start(core: CoreStart, plugins: StartDependencies): EmbeddableExplorerStart {
+    const helloWorldAction = new HelloWorldAction(core.overlays);
+    const sayHelloAction = new SayHelloAction(alert);
+    const sendMessageAction = new SendMessageAction(core.overlays);
+    const helloWorldEmbeddableFactory = new HelloWorldEmbeddableFactory();
+    const contactCardEmbeddableFactory = new ContactCardEmbeddableFactory(
+      {},
+      start.executeTriggerActions,
+      core.overlays
+    );
+
+    start.registerAction(helloWorldAction);
```

Comment:
You probably want to `start.` 👉 `plugins.embeddable.`.

Reply:
If you mean linter error, it is not configured for functional tests: https://github.com/elastic/kibana/blob/master/.eslintrc.js#L160-L174

Reply:
I changed that in this PR - https://github.com/elastic/kibana/pull/44110/files#diff-e4403a877d80de653400d88d85e4801aR163

It's just not a import via `ui/` the import was relative.  I thought there was no `start` object that's why I was surprised it didn't throw an undefined error, but I see now it was imported directly above.  Fixed!